test/Microsoft.Health.Dicom.Web.Tests.E2E/Rest/StoreTransactionTests.cs

```diff
@@ -452,6 +452,17 @@ public async Task GivenInstanceWithPatientIDWithInvalidChars_WhenStoreInstanceWi
         failedAttributesSequence.Items[0].GetString(DicomTag.ErrorComment));
     }
 
+    [Fact]
+    public async Task GivenAValidUpdateAttributeRequest_WhenUpdating_ThenShouldSucceed()
```

Comment:
Could we add the same test in `DataPartitionEnabledTests`?

Reply:
Done